server/lib/realtime/replication.ex

```diff
@@ -128,7 +128,8 @@ defmodule Realtime.Replication do
           table: name,
           columns: columns,
           record: data,
-          commit_timestamp: commit_timestamp
+          commit_timestamp: commit_timestamp,
+          is_rls_enabled: false
```

Comment:
backward compatibility: setting `is_rls_enabled` to false b/c it defaults to `true` for any `Record` struct